package.json

```diff
@@ -27,7 +27,7 @@
     "lodash.isfunction": "^3.0.8"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0"
+    "react": "^0.14.0 || ^15.0.0 || ^16.0.0"
```

Comment:
@pgoldrbx @brokentone 

can we remove 14 and 15?

Reply:
FOR SURE on 14. Seems like this is cross compatible with 15/16 through, right?

I believe that we're considering peer dep major version removals breaking though... so if we're going to release a breaking change, perhaps we do both as a major?

Reply:
What if we add 16 for now, and remove 14 in a separate breaking change?

Reply:
If you insist, the previous commit on this project was a bump to `1.0.0`. I don't mind adding another!